src/rust/ensogl/lib/components/src/toggle_button.rs

```diff
@@ -202,26 +205,30 @@ impl<Shape:ColorableShape+'static> ToggleButton<Shape>{
 
              // === Input Processing ===
 
+            toggle <- any(...);
```

Comment:
I believe having this kind of event is discouraged as it breaks event flow. Instead, there should be a `toggle <- any(...)` that is triggered on any required toggle.

Reply:
I would also go with `toggle <- any(&frp.toggle,&icon.mouse_down);` as it's easier to read the FRP flow then